composer.json

```
@@ -27,11 +27,14 @@
         "sort-packages": true
     },
     "extra": {
+        "branch-alias": {
+            "dev-master": "2.1.x-dev"
+        },
         "class": "Typo3Contentblocks\\ComposerPlugin\\Plugin"
     },
     "autoload": {
         "psr-4": {
-            "Typo3Contentblocks\\ComposerPlugin\\": "src/"
+            "Typo3Contentblocks\\ComposerPlugin\\": "src"
```

```suggestion
            "Typo3Contentblocks\\ComposerPlugin\\": "src/"
```
I'd like to keep that in the suggested format https://getcomposer.org/doc/04-schema.md#psr-4